crates/core/src/db/datastore/locking_tx_datastore/mod.rs

```diff
@@ -669,15 +652,16 @@ impl Inner {
             min_value: seq.min_value.unwrap_or(1),
             max_value: seq.max_value.unwrap_or(i128::MAX),
         };
-        let row = (&sequence_row).into();
+        let row = sequence_row.into();
         let result = self.insert(ST_SEQUENCES_ID, row)?;
-        let sequence_row = StSequenceRow::try_from(&result)?;
+        // TODO(centril): `result` is already owned, so pass that it.
+        let sequence_row = StSequenceRow::try_from(&result)?.to_owned();
         let sequence_id = SequenceId(sequence_row.sequence_id);
 
-        let schema = (&sequence_row).into();
+        let schema = sequence_row.into();
         self.sequence_state.sequences.insert(sequence_id, Sequence::new(schema));
 
-        log::trace!("SEQUENCE CREATED: {}", seq.sequence_name);
+        log::trace!("SEQUENCE CREATED: id = {}", sequence_id);
```

Comment:
Why did this change from giving a name to an ID?

Reply:
I also just now clicked the show-me-a-few-more-lines-please github button and saw that the corresponding opening log has the id and name, so I don't think this is a big regression. 👍